src/components/Navigation/NavButton.spec.js

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import NavButton from './NavButton'
+
+describe('NavButton', () => {
+  it('renders a button', () => {
+    render(<NavButton />)
+
+    const button = screen.getByRole('button')
+    expect(button).toBeInTheDocument()
+  })
+
+  it('calls a function when button is clicked and navigation state is false', () => {
+    const mockShowNavigation = jest.fn()
+    const mockNavigation = false
+
+    render(
+      <NavButton
+        showNavigation={mockShowNavigation}
+        navigation={mockNavigation}
```

Comment:
This is a bit easier to read I think.
```suggestion
        navigation={false}
```

Reply:
Done, thanks!